lessons/sums.rb

```diff
@@ -0,0 +1,28 @@
+class Sum1
+
+    attr_accessor :total
+
+    def initialize(a, b) 
+        @total = a + b
+    end
+end
+
+class Sum2
+
+    attr_accessor :a, :b
```

Comment:
FYI: You don't actually need `attr_accessor` for `Sum2`, since we are only accessing these attributes using their instance variable name (`@a` and `@b`) inside of an instance method.